test/e2e/e2ethanos/custom_test_matchers.go

```diff
@@ -0,0 +1,16 @@
+package e2ethanos
+
+import (
+	e2emon "github.com/efficientgo/e2e/monitoring"
+)
+
+// Between is a MetricValueExpectation function for WaitSumMetrics that returns true if given single sum is between
+// the lower and upper bounds (non-inclusive, as in `lower < x < upper`).
+func Between(lower, upper float64) e2emon.MetricValueExpectation {
```

Comment:
Hmm, so the expectation is that whatever metric is being asserted would freeze at a particular value, between lower and upper bounds?

Reply:
The expectation is precisely the meaning of the mathematical expression `lower < x < upper`: `x` can be anything between `lower` and `upper`. It's effectively the same as running `Higher(lower)` and `Lower(upper)`.